### ISSUE TITLE
Info.plist issues with CFBundleVersion

### ISSUE THREAD

**Issue by elronalds** [maxogden/electron-packager#242]
When the application's Info.plist is created the CFBundleVersion is an integer type and this causes issues.  For example when clicking the About menu (with the about role that calls `orderFrontStandardAboutPanel`) in an electron.js app, the about dialog will only open on every second menu click. The version is also not shown when it eventually does open.

```
<key>CFBundleVersion</key>
<integer>100</integer>
```

The about menu starts working correctly if this is changed to a string type as shown below:

```
<key>CFBundleVersion</key>
<string>100</string>
```

I believe this should be patched to correct the issue.  Also the default blank electron.js app has the CFBundleVersion set as a string type.

**Comment by malept** [comment]
This seems reasonable, and an easy issue for a new contributor to take on.

**Comment by malept** [comment]
Fixed by #250.
